src/harness/reference_models/examples/benchmark/time_dpa.py

```diff
@@ -0,0 +1,257 @@
+"""Benchmark the DPA model timing.
+
+This is a simple simulation of the DPA for purpose of estimating time to run.
+
+Usage:
+  python time_dpa.py
+
+Notes:
+  - multiprocessing facility not tested on Windows (use 1 process if issues)
+  - if warning reported on cached tiles swapping, increase the cache size.
+"""
+# TODO(sbdt): add reporting for memory usage
+from collections import namedtuple
+import multiprocessing
+import time
+
+import cartopy.crs as ccrs
+import matplotlib.pyplot as plt
+import numpy as np
+
+from reference_models.move_list_calc import move_list
+from reference_models.geo import zones
```

Comment:
Somehow I am not able to find zone.py in this branch? Am I missing something?

Reply:
No it is missing from this PR.
Let me add them now, and address your other comments.

Reply:
It was done on another branch to keep this one only focused.
But you are right, better to add it here. You should see it now